tests/splinter_test.py

```diff
@@ -204,6 +204,33 @@ def test_add_two_assets_upload(self):
             self.assertEqual(assets[1]['mimetype'], u'image')
             self.assertEqual(assets[1]['duration'], settings['default_duration'])
 
+    def test_add_asset_streaming(self):
+        with Browser() as browser:
+            browser.visit(main_page_url)
+
+            wait_for_and_do(browser, '#add-asset-button', lambda btn: btn.click())
+            sleep(1)
+
+            wait_for_and_do(browser, 'input[name="uri"]', lambda field: field.fill('rtmp://controlstreams.com:1935/vtvmoviles3/vtvmoviles3'))
```

Comment:
Maybe we should install https://github.com/theintencity/rtmplite and spin that up with some static material in the unit test setup? Hopefully it's pretty easy to get going. Then we can kill it after teardown.